srcpkgs/OpenJK/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'OpenJK'
+pkgname=OpenJK
+version=0.0.20210922
+revision=1
+archs="i686 x86_64"
+_gitrev=43e9a3da555a487e23081bbd03101e7c926a1e83
+wrksrc="${pkgname}-${_gitrev}"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/opt/${pkgname} -DBuildJK2SPEngine=on -DBuildJK2SPGame=on -DBuildJK2SPRdVanilla=on"
```

Comment:
Please add a line break after ~70 chars or smth.
Also change the prefix to smth like `/usr/lib/OpenJK` instead of `/opt`, `/opt` is for all kind of weird Third Party stuff, not packages in the repo.

Reply:
Also you shouldn't have to add a  `CMAKE_BUILD_TYPE`

Reply:
is 80 characters for the line break cool?

Reply:
Yeah, 80 is also fine